packages/components/src/custom-select-control/test/index.js

```diff
@@ -48,13 +48,23 @@ const props = {
 	],
 };
 
-const ControlledCustomSelectControl = ( { options, ...restProps } ) => {
+const ControlledCustomSelectControl = ( {
+	options,
+	onChange: onChangeMock,
+	...restProps
+} ) => {
 	const [ value, setValue ] = useState( options[ 0 ] );
+	const onChange = ( changeObject ) => {
+		if ( onChangeMock ) {
+			onChangeMock( changeObject );
+		}
```

Comment:
Nit: prefer optional chaining

Reply:
Swapped to optional chaining and actually uniformed the code between v1 and legacy v2